liquibase-core/src/main/java/liquibase/snapshot/jvm/UniqueConstraintSnapshotGenerator.java

```diff
@@ -342,6 +342,14 @@ protected void addTo(DatabaseObject foundObject, DatabaseSnapshot snapshot) thro
                 columnCache = new HashMap<>();
                 snapshot.setScratchData(cacheKey, columnCache);
                 for (Map<String, ?> row : rows) {
+                    if (!row.containsKey("CONSTRAINT_CONTAINER"))
```

Comment:
I get the worry that without the constraint_container columns, other databases will hit the same bug and it will go un-noticed. But, this check will end up breaking snapshot support completely vs. being wrong. 

We should find a way to either know what databases can support the "bulk" queries either dynamically (by searching the sql for a constraint_contaier string) or hard-coding the list better.